bin/cordlr

```diff
@@ -20,17 +15,17 @@ main()
 
 function main () {
   // --help, -h, help
-  if (flags.help || flags.h || action === 'help') return help()
+  if (flags.help || flags.h || action === 'help') return require('./methods/help')()
```

Comment:
The reason i did that is because i ran into an issue where when i would run cordlr setup before the package.json was created it would crash because bin/cordlr required the start method and that required the bin/config.js file and that file only works when a package.json is already present